static/js/ranking/services/ranking.service.js

```diff
@@ -29,6 +33,56 @@
 
 
     /**
+    * @name getWorkerRankings
+    * @desc Get worker rankings.
+    * @returns {Promise}
+    * @memberOf crowdsource.ranking.services.RankingService
+    */
+    function getWorkerRankings() {
+      var settings = {
+        url: '/api/rating/workers_reviews/',
+        method: 'GET',
+      };
+      return HttpService.doRequest(settings);
+    }
+
+    function getRequesterRankings() {
+      var settings = {
+        url: '/api/rating/requesters_reviews/',
+        method: 'GET',
+      };
+      return HttpService.doRequest(settings);
+    }
+
+    function submitRating(rating, entry) {
+      var settings = {
+        url: '/api/worker-requester-rating/',
+        method: 'POST',
+        data: {
+          weight: rating,
+          type: entry.reviewType,
+          target: entry.target,
+          module: entry.module
+        }
+      };
+      return HttpService.doRequest(settings);
+    }
+
+    function updateRating(rating, entry) {
+      var settings = {
+        url: '/api/worker-requester-rating/' + entry.current_rating_id + '/',
+        method: 'PUT',
+        data: {
+          weight: rating,
```

Comment:
since you are using current_rating_id only weight is required here, you may set partial=True on update on backend